modules/apps/portal-workflow/portal-workflow-kaleo-runtime-integration-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/integration/internal/WorkflowTaskManagerImpl.java

```diff
@@ -312,15 +310,15 @@ public List<String> getNextTransitionNames(long userId, long workflowTaskId)
 
 			KaleoNode kaleoNode = kaleoTask.getKaleoNode();
 
-			return Stream.of(
-				kaleoNode.getKaleoTransitions()
-			).flatMap(
-				List::parallelStream
```

Comment:
@tinatian Hi Tina,
Do I need to do anything different to convert parallel stream?
Online information seems to suggests that for loop is faster than both serial and parallel stream

Reply:
https://blogs.oracle.com/javamagazine/post/java-parallel-streams-performance-benchmark